src/topology/connected.lean

```diff
@@ -198,25 +198,48 @@ def connected_component_in (F : set α) (x : F) : set α := coe '' (connected_co
 theorem mem_connected_component {x : α} : x ∈ connected_component x :=
 mem_sUnion_of_mem (mem_singleton x) ⟨is_connected_singleton.is_preconnected, mem_singleton x⟩
 
+theorem is_preconnected_connected_component {x : α} : is_preconnected (connected_component x) :=
+is_preconnected_sUnion x _ (λ _, and.right) (λ _, and.left)
+
 theorem is_connected_connected_component {x : α} : is_connected (connected_component x) :=
-⟨⟨x, mem_connected_component⟩, is_preconnected_sUnion x _ (λ _, and.right) (λ _, and.left)⟩
+⟨⟨x, mem_connected_component⟩, is_preconnected_connected_component⟩
 
-theorem subset_connected_component {x : α} {s : set α} (H1 : is_preconnected s) (H2 : x ∈ s) :
-  s ⊆ connected_component x :=
+theorem is_preconnected.subset_connected_component {x : α} {s : set α}
+  (H1 : is_preconnected s) (H2 : x ∈ s) : s ⊆ connected_component x :=
 λ z hz, mem_sUnion_of_mem hz ⟨H1, H2⟩
 
+theorem is_connected.subset_connected_component {x : α} {s : set α}
```

Comment:
This is strictly weaker than `is_preconnected.subset_connected_component`, but also makes it easier to use dot notation. (Without this, you have to convert from `is_connected` to `is_preconnected` which results in ugly `.2.subset_connected_component`s.)